nettrine/app/Model/Database/Fixtures/BooksFixture.php

```diff
@@ -15,14 +17,50 @@ class BooksFixture implements FixtureInterface, OrderedFixtureInterface
 	 */
 	public function load(ObjectManager $manager): void
 	{
+		$connection = $manager->getConnection();
+
+		$sql = "ALTER TABLE ext_translations AUTO_INCREMENT = 1;";
```

Comment:
K cemu je tu tohle?

Reply:
mám to smazat?

Reply:
Smaz to prosim. Nechme to jednoduche.

Reply:
smazat jen v readme? Nebo odstanit i soubory okolo dockeru?

Reply:
I ty soubory prosim, dikes.